### ISSUE TITLE
Feature Request: Dev Events also add remaining length in millimeters.

### ISSUE THREAD

**Issue by Terandium** [OllisGit/OctoPrint-SpoolManager#306]
Heyy,
Love the usage of Octoprint events so I can hook into it.
However, right now all it provides is remainingWeight (which should be remaining_weight btw, JSON uses snake_casing)
In order to transfer grams to millimeters I need to know the density of the filament, which it doesn't say.
I only print in PLA so it is fine for me to hardcode it that way, but doing it properly would be nice.

I need (milli)meters instead of grams because what I am trying to hook in to only gives the filament length instead of weight.

TL;DR: Add the remaining (milli)meters of filament to events
